concepts/get-started-transforming.md

```diff
@@ -0,0 +1,399 @@
+# Get started transforming streams with map, pluck, and mapTo
+
+When working with observables one of the most common use cases you will
+encounter is the need to transform a stream of some value type into a stream of
+another value type. For instance, you may have an observable of click events
+that you wish to transform into an observable of objects containing just the
+`clientX` and `clientY` coordinates. Or maybe you need to extract a value from a
+stream of input events to perform a calculation or initiate a request. Or
+perhaps you just want to extract a single property from an object, like a key
+code, to perform another action down the (pipe)line. The scenarios for
+transforming streams are endless.
+
+In this article, we are going to learn about the most common operator used to
+transform streams, the `map` operator. We will start by taking a look at
+`Array.map` to build a general understanding of the `map` operation. Next, we
+will explore how we can apply this approach to observables by using the RxJS
+`map` operator. Finally, we will check out a few helper operators that can be
+used in place of `map` should the right scenario present itself, exploring
+common use cases along the way. Let's get started!
+
```

Comment:
Maybe worth mentioning here that there are more operators that work similarly to JS array functions like filter or reduce?

Reply:
Good idea, I'm going to add this as a tip callout in the next section. 👍